lib/internal/cluster/master.js

```diff
@@ -41,15 +42,15 @@ if (schedulingPolicy === undefined) {
 
 cluster.schedulingPolicy = schedulingPolicy;
 
-cluster.setupMaster = function(options) {
+cluster.setupMaster = function(options = {}) {
```

Comment:
Not sure why its backwards incompatible, but it is unnecessary, as is `util._extend(settings, options || {})`. `_extend()` ignores non-object second args. Default here should be removed, and the `|| {}` that is removed below can stay removed.